.mergify.yml

```diff
@@ -0,0 +1,37 @@
+pull_request_rules:
+  - name: automatic merge
+    conditions:
+      - and: &base_checks
+        - base=master
+        - -label~=^acceptance-tests-needed|not-ready
```

Comment:
Sorry i just notice that label is already applied. But i think the WIP is missing.

Reply:
Sorry I don't know what you mean

Reply:
Some people still use `[WIP] ` as the prefix either in pull request title or even in commit message.

Reply:
I'd ignore commit message and match only label.

Reply:
some people do not have merge permissions to use labels, the check for WIP in the title still apply here, right?

Reply:
yes, WIP is handled by https://github.com/apps/wip and any WIP in the PR subject or git commit message will still block merge

Reply:
What about the `WIP` label? I found that [apps/wip](https://github.com/apps/wip) checks labels only in some kind of 'PRO' version..

Reply:
ok, included "wip" label.